files/image_config/platform/rc.local

```diff
@@ -319,6 +319,13 @@ if [ -f $FIRST_BOOT_FILE ]; then
         mv /host/grub.cfg /host/grub/grub.cfg
     fi
 
+    if `grep crashkernel=256M /proc/cmdline > /dev/null 2>&1`;
+    then
+	kdump-config unload
+	kdump-config symlinks `uname -r`
+	kdump-config load
+    fi
```

Comment:
Follow code format nearby.

Reply:
I notice you mix space with tab. Please only use space in this file and follow code format nearby.

---
In reply to: [331242454](https://github.com/Azure/sonic-buildimage/pull/3276#discussion_r331242454) [](ancestors = 331242454)